### DISCUSSION TITLE
How to stream the camera and mic in the background?

### DISCUSSION THREAD

**Discussion by Serg-Pogrebnyak** [ant-media/Ant-Media-Server#6582] [Q&A]
I built the test application following this guide - https://antmedia.io/build-and-try-webrtc-ios-app-for-dummies/, but when I moved to another application my camera failed and I had only an audio stream, but after some time and audio connect was interrupted. How I can continue to stream the mic and camera when I'm switching to another app?

**Comment by Serg-Pogrebnyak** [comment]
it's impossible, because:
> Background Apps May Not Execute Commands on the Graphics Hardware

https://developer.apple.com/library/archive/documentation/3DDrawing/Conceptual/OpenGLES_ProgrammingGuide/ImplementingaMultitasking-awareOpenGLESApplication/ImplementingaMultitasking-awareOpenGLESApplication.html

**Reply by Mohit-3196** [reply]
Thank you @Serg-Pogrebnyak for your questions and comments.
